src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/ViewModels/PublicationSummaryViewModel.cs

```diff
@@ -0,0 +1,14 @@
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.ViewModels;
+
+public record PublicationSummaryViewModel
```

Comment:
Shouldn't this go in `PublicationViewModels`? 

Doesn't seem like we need `PublicationListViewModel` either, as this should just be the same as `PublicationSummaryViewModel`

Reply:
Yeah! I was meant to do this actually but forgot. Good spot. 

I've moved `PublicationSummaryViewModel` into `PublicationViewModels.cs`

However, in regards to the other comment, I thought we might want to keep them separate because they may divert with time? I know now they are identical view models, but wasn't sure if they'd remain like that?

Reply:
discussed and agreed to just use the 1 model for now :)